NitroxClient/Debuggers/SceneDebugger.cs

```diff
@@ -293,7 +295,18 @@ private void RenderTabHierarchy()
 
                     if (string.IsNullOrEmpty(gameObjectSearchPatternInvalidMessage))
                     {
+                        Type type = AppDomain.CurrentDomain.GetAssemblies()
+                        .Select(a => a.GetType(gameObjectSearch))
```

Comment:
Case insensitive search might be good here so you don't have to type the name exactly.